iphone/TitaniumKit/TitaniumKit/Sources/API/TiViewProxy.m

```diff
@@ -166,8 +166,6 @@ - (BOOL)belongsToContext:(id<TiEvaluator>)context
 
 - (void)add:(id)arg
 {
-#if IS_XCODE_9
-#ifdef USE_TI_UIWINDOW
```

Comment:
Can you explain why these could all be removed?

Reply:
IS_XCODE_9 and  USE_TI_UIWINDOW macro is not available at Titanium framework. It was only ensuring to unnecessarily code compilation